projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -456,5 +462,16 @@
     },
     "dynamicNavigation": {
         "couldNotResolveView": "Ein Fehler ist aufgetreten während des Ansichtauflösungsprocess. Der Ansicht kann nicht angezeigt werden"
+    },
+    "impersonation": {
+        "user": {
```

Comment:
I am unsure about these translations. I will consult this with someone and report back later...

Reply:
success: "Benutzer wird jetzt vertreten"
already: "Für den Benutzer existiert bereits eine Vertretung"
loggedIn: "Benutzer ist bereits angemeldet"